lib/outputs/outputs.py

```diff
@@ -3,20 +3,19 @@
 from colorama import Fore, Style
 
 from config.config import csv_file_suffix
-from lib.check.models import Organizations_Info
 from lib.outputs.models import Check_Output_CSV
 from lib.utils.utils import file_exists, open_file
 
 
-def report(check_findings, output_options, audit_info, organizations_info):
+def report(check_findings, output_options, audit_info):
     check_findings.sort(key=lambda x: x.region)
 
     csv_fields = []
     # check output options
     file_descriptors = {}
     if output_options.output_modes:
         if "csv" in output_options.output_modes:
-            csv_fields = generate_csv_fields()
+            csv_fields = generate_csv_fields(audit_info.organizations_metadata)
```

Comment:
Please, I think it is not a good practice to have variable fields in the csv file. 
If we do not fill the fields that fields must be empty